djhtml/modes.py

```diff
@@ -549,6 +556,8 @@ def create_token(self, raw_token, src, line):
                 token = Token.Text(raw_token, mode=InsideHTMLTag, **self.offsets)
         elif not self.inside_attr and raw_token == "/>":
             token, mode = Token.Text(raw_token, mode=DjHTML), self.return_mode
+            if not self.absolute:
```

Comment:
This is replaceable by an if that return `Text` or `Close` depending on `self.absolute`.